docs/release_notes/v1.10.7.md

```diff
@@ -17,3 +17,29 @@ The latest versions (1.38.X) of the shopify/sarama kafka library introduced this
 ### Solution
 
 Downgrading the shopify/sarama library to release 1.37.2 resolves this problem. An issue with the maintainers of the sarama library has been created for further investigation.
+
+## Fixed an issue with HTTP URL routing in Dapr sidecar
+
+### Problem
+
+When using the Dapr sidecar HTTP API, some crafted HTTP URLs where not being routed to
+the correct handler, for example, when prefixing the URL path with duplicated
+slashes. i.e. `//v1.0/actors/DemoActor/1234/method/SetReminder`.
+
+### Impact
+
+Requests which did not have perfect paths according to the API may be routed to
+the wrong API handler, and typically return some kind of error.
+
+### Root cause
+
+The root HTTP request handler was not normalizing the request's HTTP path before
```

Comment:
I would keep it less specific.

> The HTTP router inside Dapr was "normalizing" the request's HTTP path only after evaluating the route, causing routing errors with un-normalized paths.

Reply:
:+1: 

Btw, *normalization* is a well defined term that doesn't need quoting. https://datatracker.ietf.org/doc/html/rfc3986#section-6